test/support/fixtures/vcr_cassettes/scrapers/search_tv_failed.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
do we use this file somewhere @nvminhtue if not can we delete it

Reply:
@andyduong1920 this file is automatically generated through ```use_cassette```